app/assets/javascripts/client/inventories/AddInventoryUsersCtrl.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').controller('AddInventoryUsersCtrl', AddInventoryUsersCtrl);
+
+  AddInventoryUsersCtrl.$inject = ['$scope', 'InventoryInvitable', 'InventoryParticipant'];
+  function AddInventoryUsersCtrl($scope, InventoryInvitable, InventoryParticipant) {
+    var vm = this;
+    vm.reloadInvitables = function() {
+      vm.invitables = InventoryInvitable.list({ inventory_id: $scope.inventoryId });
+    }
+    vm.reloadInvitables();
+
+    vm.addUser = function(user) {
+      InventoryParticipant.create({ inventory_id: $scope.inventoryId }, { user_id: user.id }).$promise.then(function() {
```

Comment:
I remember us discussing this the other day - it was either a matter of us passing this in as an attribute value or pulling it off of `$stateParams`.

At the time I said it was essentially the same, but when I read the code, I believe I prefer the `$stateParams` path better.

Simply put, we both agree that the only way this will be invoked is if it falls on a path parameter that contains the inventory ID.  Since that information is part of the route, we should be leveraging it.

It makes the code simpler in that:

 - We no longer need to include `$scope`
 - We use a more conventional and standard approach to pulling the path parameter

It makes the tests simpler in that we can supply our own take `$stateParams` without the need to run a digest cycle.